scripts/issue/561/claim_airdrops.py

```diff
@@ -0,0 +1,38 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+from brownie import interface
+import json
+
+
+"""
+eligble:
+treasury voter: 0xA9ed98B5Fb8428d68664f3C5027c62A10d45826b
+ibbtc msig: 0xB76782B51BFf9C27bA69C77027e20Abd92Bcf3a8
+ibbtc peak (todo: redirect): 0x41671BA1abcbA387b9b2B752c205e22e916BE6e3
```

Comment:
there is no proof json for this address, does this mean the redirect was successful and its claimable amount is indeed 0?

Reply:
i think for the peak was redirected to the treasury_voter. [ref](https://github.com/aurafinance/aura-token-allocation/blob/6e95bc02fe6a1d401a7d4fbdf8634f49e21f6738/redirection/badger.json#L3)

Reply:
damn if so easy why didnt we redir all to voter..?